android/BOINC/app/src/main/res/values-en/strings.xml

```diff
@@ -349,7 +349,7 @@
     <string name="about_name">BOINC</string>
     <string name="about_version">Version %1$s</string>
     <string name="about_name_long">Berkeley Open Infrastructure for Network Computing</string>
-    <string name="about_copyright">\u00A9 2003–2016 University of California, Berkeley.</string>
+    <string name="about_copyright">\u00A9 2003–2020 University of California, Berkeley.</string>
```

Comment:
Please remove this change since it is in translatable files that are managed by Transifex

Reply:
If I remove this change, It will show 2016 in the about.